static/src/js/containers/MetricsEventsContainer/MetricsEventsContainer.jsx

```diff
@@ -31,7 +31,7 @@ export class MetricsEventsContainer extends Component {
 
     if (!clickableParent) return
 
-    const title = target.title || target.text
+    const title = target.title || target.text || target.name || target.innerText
```

Comment:
Updated this to fix some of the metrics from default click having an `undefined` value. Others have issues due to` event.stopPropogation` being called spun off separate ticket for that